app/src/main/res/values/strings.xml

```diff
@@ -15,7 +17,19 @@
     <string name="genres_edm">EDM</string>
     <string name="genres_electronic">Electronic</string>
 
+
     <string name="image_track">image_track</string>
     <string name="image_more_info">moreinfo</string>
+    <string name="add_album">Add Album</string>
+    <string name="name_album">Name Album</string>
+    <string name="err_name_exist">Album name  is exist !!</string>
+    <string name="err_name_null">Name album is empty</string>
+    <string name="song">Songs</string>
+    <string name="delete">Delete</string>
+    <string name="rename">Rename</string>
+    <string name="rename_album">Rename Album</string>
+    <string name="ok">Ok</string>
```

Comment:
WARNING: [Typos] "Ok" is usually capitalized as "OK"
 This check looks through the string definitions, and if it finds any words that look like likely misspellings, they are flagged.